examples/function_todomvc/src/components/entry.rs

```diff
@@ -48,6 +48,15 @@ pub fn entry(props: &EntryProps) -> Html {
         move |_| onremove.emit(id)
     };
 
+    let onedit = {
+        let onedit = props.onedit.clone();
+        let edit_toggle = edit_toggle.clone();
+        move |value| {
+            edit_toggle.clone().toggle();
```

Comment:
Nit: pretty sure that you don't need this clone but clippy isn't complaining so it's fine

Reply:
Unfortunately, both clones appear to be necessary.  `.toggle()` takes ownership, so without cloning the closure is only `FnOnce`.

I tried a few different options to reduce it down to a single clone, but I couldn't find a way to do so that satisfied the borrowing or ownership rules.